cmake/FindSIP.py

```diff
@@ -48,12 +50,15 @@
     else:
         print("sip_module_dir:%s" % sipcfg.sip_mod_dir)
 else:  # Code for SIP v5
+    import shutil
+    sip_bin = shutil.which("sip5")
+    if not sip_bin:
+        sys.exit(1)
```

Comment:
Would it make sense to print an error message here or is that covered already by cmake?
Can sip_bin be manually specified somewhere as a cmake variable? This code assumes it's on system path.

Reply:
Missing sip5 will result in an error "SIP not found". It could be more verbose but then it has to be changed in FindSIP.cmake. FindSIP.cmake is doing really lousy work on detecting if values coming back from FindSIP.py make any sense at all thus sys.exit(1) is the only safe option. Fiddling with cmake files is a bit out of my league and I lack free time to learn cmake.
Threat this PR not as an end but as a step into right direction.

Reply:
Just wondering if a simple print or print to stderr in python would help?

And for the other question it's likely that this script is not even called if a sip binary has already been specified, so no worries, that works for me.